docker/Dockerfile-sbt

```diff
@@ -1,6 +1,6 @@
 FROM jenkins/jnlp-slave
 USER root
-ENV AWS_DEFAULT_REGION eu-west-2
+ENV AWS_DEFAULT_REGION eu-west-1
```

Comment:
Can we keep this in eu-west-2 (especially since the commit message is "Update the region to eu-west-2"? If not, it's worth a comment explaining why.

Reply:
Yeah we are, this was for testing on the dri account because we ran out of network interfaces.